torch/fx/graph.py

```diff
@@ -627,11 +628,52 @@ def delete_unused_values(user : Node):
             else:
                 body.append('\n')
 
+        def preprocess_values(a : Argument):
```

Comment:
this is a massive copy/paste of map_arg, doesn't seem sustainable.

Reply:
This is different from `map_arg` in that the configurable behavior applies to data types other than `Node`. Not sure how to generalize `map_arg` to accept that except getting into weird generic type territory